docs/guide/docs/developers/tutorials.md

```diff
@@ -348,3 +348,60 @@ module.exports = {
 ```
 
 Notice in the above example that via the `web-style` key we are changing the direction in which the text is written, making it right-to-left for Arabic
+
+## Calling an API in a Custom Action
+
+One of the most popular use-case for Actions is to call an API, get some data and use it in your flow. That's what we're going to demonstrate here.
+
+We're going to use `axios` as http client because its already a Botpress dependency. See our [Custom Code](https://botpress.io/docs/build/code/) section to learn more about how dependencies work in Actions.
+
+Start by creating a new javascript file in `/data/bots/<your_bot>/actions/getQuote.js`. Then copy the following code:
+
+```javascript
+const axios = require('axios')
+
+const getQuote = async () => {
+  // We declare a new axios instance
+  const client = axios.create({
+    baseURL: 'https://opinionated-quotes-api.gigalixirapp.com/v1'
+  })
+
+  // And we get a new quote
+  const quote = await client.get('/quotes').then(res => res.data.quotes[0].quote)
+}
```

Comment:
quote should actually be returned here :-p I think it could be a bit clearer with the link to the bot reply part